docs/pages/ui-programming/react-native-toast.md

```diff
@@ -0,0 +1,111 @@
+---
+title: How to display a toast in React Native
+sidebar_title: Displaying toast messages
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+## What is a toast?
+
+According to the [Android Developers
+Documentation](https://developer.android.com/guide/topics/ui/notifiers/toasts): "A toast provides
+simple feedback about an operation in a small popup. It only fills the amount of space required for
+the message and the current activity remains visible and interactive. Toasts automatically disappear
+after a timeout".
+
+To present a toast, we recommend two solutions: an API from React Native (RN) itself and a library
+maintained by the RN community.
+
+## `ToastAndroid`
+
+Toasts are a native feature of Android, but iOS doesn't have this by default. If you only need
+toasts on Android, you can use the [`ToastAndroid`](https://reactnative.dev/docs/toastandroid) API
+provided by React Native.
+
+### Usage
+
+To show a basic toast with `ToastAndroid`, import `ToastAndroid` from the `'react-native'` package
+and call `ToastAndroid.show` with a message and duration option:
+
+```jsx
+import React from 'react';
+import { View, StyleSheet, ToastAndroid, Button, StatusBar } from 'react-native';
+
+export default function App() {
+  function showToast() {
+    ToastAndroid.show('Request sent successfully!', ToastAndroid.SHORT);
+  }
+
+  return (
+    <View style={styles.container}>
+      <Button title="Show Toast" onPress={showToast} />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    paddingTop: StatusBar.currentHeight,
+    backgroundColor: '#6638f0',
+    padding: 8,
+  },
+});
+```
+
+The code above results in this on a Pixel 3a:
+
+<ImageSpotlight style={{maxWidth: 360}} alt="Pixel 3a showing toast message in an Expo app" src="/static/images/ToastAndroid.png" />
+
+There are many other ways to configure your toast position, duration, and gravity options. Read the
+[React Native `ToastAndroid`](https://reactnative.dev/docs/toastandroid) docs to learn more.
+
+## `react-native-root-toast`
+
+Since iOS doesn't have a built-in toast feature, React Native developers have to implement their own
+cross-platform toast libraries.
+[`react-native-root-toast`](https://github.com/magicismight/react-native-root-toast) is one such
```

Comment:
Would be nice to talk to why you are recommending this module over the other community solutions